src/main/scala/bubblewrap/PageParser.scala

```diff
@@ -39,10 +43,10 @@ trait ContentType {
     "gzip/document"
   )
 
-  def isGzip(content: Content) = {
+  def isGzip(content: Content): Boolean = {
```

Comment:
Generally, you wont see functions / methods with return types explicitly provided as part of its signature in our code base. Just a convention, we follow everywhere.

Reply:
Fixed in 92fb12f.